appointment/templates/appointment/form_submission.html

```diff
@@ -1,13 +1,15 @@
 {% extends "pttrack/base-form.html" %}
 
+{% load i18n %}
+<!-- trans finished -->
 {% load bootstrap3 %}
 
 {% block title %}
 {{ note_type }} {{ patient.name }}
 {% endblock %}
 
 {% block header %}
-<h1>New {{ note_type }} </h1>
+<h1>{% trans "New" context "neuer" %} {{ note_type }} </h1>
```

Comment:
What's going on in this line? I'm surprised to see any non-base language words (in this case "neuer")... is that necessary?

Reply:
These are context based translations. You are right, there should be a better way than to write it down in german! Basically this tells the i18n script that this word "new" can have different meanings or translations in german (e.g. it could translate to "neu" oder "neuer"). I try to think about a better way to implement this.